src/components/popover-container/popover-container.component.js

```diff
@@ -77,13 +111,72 @@ const PopoverContainer = ({
 };
 
 PopoverContainer.propTypes = {
-  /** Sets the popover container dialog header name */
-  title: PropTypes.string.isRequired,
-  /** Sets the icon that opens dialog */
-  iconType: PropTypes.string.isRequired,
-  /** Sets rendering position of dialog */
+  /** A function that will render the open component
+   *
+   * `({dataElement, tabIndex, onClick, ref, ariaLabel, isOpen}) => ()`
+   *
+  */
+  renderOpenComponent: PropTypes.func,
+  /** A function that will render the close component
+   *
+   * `({dataElement, tabIndex, onClick, ref, ariaLabel, isOpen}) => ()`
+   *
+  */
+  renderCloseComponent: PropTypes.func,
+  /** If `true` the popover-container will open */
+  open: PropTypes.bool,
+  /** Sets the popover-container title displayed in the dialog */
+  title: PropTypes.string,
+  /** If `true` the popover-container will cover open button */
+  shouldCoverButton: PropTypes.bool,
+  /** Callback fires when open component is clicked */
+  onOpen: PropTypes.func,
+  /** Callback fires when close component is clicked */
+  onClose: PropTypes.func,
+  /** Sets rendering position of the popover-container */
   position: PropTypes.oneOf(['left', 'right']),
-  children: PropTypes.node
+  /** The content of the popover-container */
+  children: PropTypes.node,
+  /** The id of the element that describes the dialog */
+  ariaDescribedBy: PropTypes.string
+};
+
+PopoverContainer.defaultProps = {
+  position: 'right',
+  shouldCoverButton: false,
+  renderOpenComponent: ({
+    // eslint-disable-next-line react/prop-types
+    tabIndex, onClick, dataElement, ref, ariaLabel
+  }) => (
+    <PopoverContainerOpenIcon
+      tabIndex={ tabIndex }
+      onAction={ onClick }
+      data-element={ dataElement }
+      ref={ ref }
+      aria-label={ ariaLabel }
+      aria-haspopup
+    >
+      <Icon
+        data-element='popover-container-open-component'
+        type='settings'
+      />
+    </PopoverContainerOpenIcon>
+  ),
+  renderCloseComponent: ({
+    // eslint-disable-next-line react/prop-types
+    dataElement, tabIndex, onClick, ref, ariaLabel
+  }) => (
+    <PopoverContainerCloseIcon
+      data-element={ dataElement }
+      type='close'
```

Comment:
```suggestion
      type='close'
```

I don't think this is used anywhere.